src/templates/can-rest-model/can-rest-model-list-short.js

```diff
@@ -0,0 +1,3 @@
+import transformer from './can-rest-model-map-short';
+
+export default transformer;
```

Comment:
Can you either add a comment explaining what is going on here or just do this in the `build/transforms.json`?